packages/catalog-process/src/services/catalogService.ts

```diff
@@ -456,7 +457,7 @@ export function catalogServiceBuilder(
 
     async updateEService(
       eserviceId: EServiceId,
-      eserviceSeed: ApiEServiceSeed,
```

Comment:
Bug fix. The endpoint was using the same seed as the createEService, even though the api spec defines two different schemas. The bug had no impact just because the two schemas were equal.